src/core/border_router/routing_manager.hpp

```diff
@@ -715,6 +715,8 @@ class RoutingManager : public InstanceLocator
         void HandleRouterTimer(void);
 
     private:
+        static constexpr uint32_t kUsableOnLinkPrefixMinPreferredLifetime = 1800; // In sec.
```

Comment:
nit: how about rename as `Favored`?
```suggestion
        static constexpr uint32_t kFavoredOnLinkPrefixMinPreferredLifetime = 1800; // In sec.
```

Reply:
Changed to "favored" in new push.